core-bundle/src/Resources/contao/templates/frontend/fe_page.html5

```diff
@@ -21,7 +21,7 @@
   <?php $this->endblock(); ?>
 
 </head>
-<body id="top"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?><?php if ($this->onload): ?> onload="<?= $this->onload ?>"<?php endif; ?> itemscope itemtype="http://schema.org/WebPage">
+<body id="top"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?><?php if ($this->onload): ?> onload="<?= $this->onload ?>"<?php endif; ?> itemscope itemtype="http://schema.org/WebPage"<?php if ($this->attributes): ?> <?= $this->attributes ?><?php endif; ?>>
```

Comment:
```suggestion
<body id="top"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?><?php if ($this->onload): ?> onload="<?= $this->onload ?>"<?php endif; ?> itemscope itemtype="http://schema.org/WebPage"<?= $this->attributes ?>>
```
I think it should be without a space, as the same concept is used in a few other templates already.

Reply:
I changed it in my latest commit.